packages/graph/hash_graph/lib/graph/src/store/postgres/context/ontology.rs

```diff
@@ -18,7 +18,6 @@ pub struct OntologyRecord<T> {
     pub owned_by_id: OwnedById,
     pub created_by_id: CreatedById,
     pub updated_by_id: UpdatedById,
-    pub removed_by_id: Option<RemovedById>,
```

Comment:
I saw no reason for us to have this, so I removed it. The commit is pretty self-contained so we can revert if we'd prefer.

I think we want to go the way we went for entities, which would mean there is no `removedById` just `updatedById`.